Docs/Configuration.tex

```diff
@@ -5485,23 +5487,23 @@ \subsection{SMBIOS Properties}\label{platforminfosmbios}
 \item
   \texttt{PlatformFeature}\\
   \textbf{Type}: \texttt{plist\ integer}, 32-bit\\
-  \textbf{Failsafe}: \texttt{0xFFFFFFFF}\\
+  \textbf{Failsafe}: \texttt{0xFFFFFFFF} (OEM specified on Apple hardware, do not provide the table otherwise)\\
```

Comment:
Failsafe is one single value. Please provide instructions below.

Reply:
Again my reason for rearranging the layout was to match what @dakanji has been doing in his current PR. Will revert as requested.